pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.deque</groupId>
     <artifactId>axe-selenium</artifactId>
-    <version>3.0</version>
+    <version>3.1.0</version>
```

Comment:
Please do not make changes to the version number. This is something we'll do on our own.